src/scss/_layout-modern.scss

```diff
@@ -12,16 +12,26 @@
   }
 
   .ucf-event-title-wrapper {
-    margin-bottom: .25rem;
+    margin-bottom: .4rem;
   }
 
   .ucf-event-title {
     color: $white;
+    font-size: 1.2rem;
     font-weight: 700;
   }
 
-  .ucf-event-location {
-    color: $white;
+  .ucf-event-description-wrapper {
+    font-size: .9rem;
+
+    a {
+      color: $white;
```

Comment:
If someone tries to use this layout on a plain, white page out-of-the-box, they won't be able to see any of the white text, and the gold text will be inaccessible.  I think that scenario is way too likely with sites that _aren't_ the main site to ignore.  We also don't set text colors in other default news or event layouts.